### ISSUE TITLE
Fails when live video encountered 

### ISSUE THREAD

**Issue by beardyjay** [woefe/ytcc#12]
When running `ytcc -ud` and a current subscription is live streaming you get a repeating error and ytcc never exits. I think this is similar to #9.

In this example, I was trying to download the latest videos from the channel Jupiter Broadcasting while they were live streaming. The last line in the error would just repeat until the process was killed.

Edit: This might be an issue with youtube-dl?

**Error output**

> [youtube] eU4NO_DWvs0: Downloading webpage
> [youtube] eU4NO_DWvs0: Downloading video info webpage
> [youtube] eU4NO_DWvs0: Downloading m3u8 information
> [youtube] eU4NO_DWvs0: Downloading MPD manifest
> [youtube] eU4NO_DWvs0: Downloading MPD manifest
> [download] Destination: /data/videos/youtube/jblive.tv Year Review Holiday Stream 🍻.mp4
> ffmpeg version n4.1 Copyright (c) 2000-2018 the FFmpeg developers
>   built with gcc 8.2.1 (GCC) 20180831
>   configuration: --prefix=/usr --disable-debug --disable-static --disable-stripping --enable-fontconfig --enable-gmp --enable-gnutls --enable-gpl --enable-ladspa --enable-libaom --enable-libass --enable-libbluray --enable-libdrm --enable-libfreetype --enable-libfribidi --enable-libgsm --enable-libiec61883 --enable-libjack --enable-libmodplug --enable-libmp3lame --enable-libopencore_amrnb --enable-libopencore_amrwb --enable-libopenjpeg --enable-libopus --enable-libpulse --enable-libsoxr --enable-libspeex --enable-libssh --enable-libtheora --enable-libv4l2 --enable-libvidstab --enable-libvorbis --enable-libvpx --enable-libwebp --enable-libx264 --enable-libx265 --enable-libxcb --enable-libxml2 --enable-libxvid --enable-nvdec --enable-nvenc --enable-omx --enable-shared --enable-version3
>   libavutil      56. 22.100 / 56. 22.100
>   libavcodec     58. 35.100 / 58. 35.100
>   libavformat    58. 20.100 / 58. 20.100
>   libavdevice    58.  5.100 / 58.  5.100
>   libavfilter     7. 40.101 /  7. 40.101
>   libswscale      5.  3.100 /  5.  3.100
>   libswresample   3.  3.100 /  3.  3.100
>   libpostproc    55.  3.100 / 55.  3.100
> [hls,applehttp @ 0x561ee0358040] Opening 'https://r4---sn-aigl6ney.googlevideo.com/videoplayback/id/eU4NO_DWvs0.0/itag/95/source/yt_live_broadcast/requiressl/yes/ratebypass/yes/live/1/cmbypass/yes/goi/160/sgoap/gir%3Dyes%3Bitag%3D140/sgovp/gir%3Dyes%3Bitag%3D136/hls_chunk_host/r4---sn-aigl6ney.googlevideo.com/playlist_type/DVR/gcr/gb/ei/gG4nXMzFB83dxwKn0oGABQ/initcwndbps/19690/mm/32/mn/sn-aigl6ney/ms/lv/mv/m/pl/32/keepalive/yes/mt/1546087976/disable_polymer/true/ip/2a01:4b00:ea1a:4c00:9ec3:fc64:f8ee:a574/ipbits/0/expire/1546109665/sparams/ip,ipbits,expire,id,itag,source,requiressl,ratebypass,live,cmbypass,goi,sgoap,sgovp,hls_chunk_host,playlist_type,gcr,ei,initcwndbps,mm,mn,ms,mv,pl/signature/6D0B12AE81C73511DF5CA55820F11E02EFE5D92F.897208F439995672DE862359287BEC44004CB829/key/dg_yt0/playlist/index.m3u8/sq/115064/goap/clen%3D48404%3Blmt%3D1545857920282089/govp/clen%3D42372%3Blmt%3D1545857920282091/dur/2.000/file/seg.ts' for reading

**YT Channel**: https://www.youtube.com/user/jupiterbroadcasting
**Live stream URL** (at the time): https://www.youtube.com/watch?v=eU4NO_DWvs0

**bug-report-info**

```
ytcc --bug-report-info
---ytcc version---
1.7.3

---youtube-dl version---
2018.12.17

---feedparser version---
5.2.1

---lxml version---
4.2.5

---dateutil version---
2.7.5

---python version---
3.7.1 (default, Oct 22 2018, 10:41:28) 
[GCC 8.2.1 20180831]

---mpv version---
mpv 0.29.1 Copyright © 2000-2018 mpv/MPlayer/mplayer2 projects
 built on Sun Dec  9 16:21:27 CET 2018
ffmpeg library versions:
   libavutil       56.22.100
   libavcodec      58.35.100
   libavformat     58.20.100
   libswscale      5.3.100
   libavfilter     7.40.101
   libswresample   3.3.100
ffmpeg version: n4.1


---config dump---
[YTCC]
dbpath = ~/.local/share/ytcc/ytcc.db
downloaddir = /data/videos/youtube
mpvflags = --really-quiet --ytdl --ytdl-format=bestvideo[height<=?1080]+bestaudio/best
ytdloutputtemplate = %(title)s.%(ext)s

[youtube-dl]
format = bestvideo[height<=?1080]+bestaudio/best
outputtemplate = %(title)s.%(ext)s
loglevel = normal
ratelimit = 0
retries = 0
subtitles = off
thumbnail = on

[quickselect]
enabled = yes
ask = yes
alphabet = sdfervghnuiojkl

[TableFormat]
id = on
date = off
channel = on
title = on
url = off
watched = off
```

**Comment by woefe** [comment]
I don't think this is an error. youtube-dl is probably downloading the live stream and will stop as soon as the live stream ends. It is also not really related to #9, because #9 was about placeholder videos  / premieres.  

What's your opinion on adding an option to skip live videos? I think it is doable and would fix this issue.

**Comment by woefe** [comment]
Fixed in 303533b. The `skiplivestream` option is enabled by default and cannot be enabled/disabled with commandline parameters. It can only be disabled in the config file
